probe/cobra.go

```diff
@@ -0,0 +1,35 @@
+package probe
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func CobraCommand() *cobra.Command {
+	var (
+		probeOptions Options
+	)
+
+	prb := &cobra.Command{
+		Use:   "probe",
+		Short: "Check the liveness or readiness of a locally-running server",
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			if !probeOptions.IsValid() {
+				return fmt.Errorf("some options are not valid")
```

Comment:
This error message is poor.  Can you change IsValid() to return an `error` or add `func (o *Options) Validate() error` so that a real message can be returned?

Reply:
Sure, added `Validate()`. Done.